dotnet/src/Microsoft.AutoGen/Agents/Agent.cs

```diff
@@ -345,4 +343,9 @@ public async ValueTask PublishEventAsync(string topic, IMessage evt, Cancellatio
     {
         await PublishEventAsync(evt.ToCloudEvent(topic), cancellationToken).ConfigureAwait(false);
     }
+
+    public void Dispose()
```

Comment:
Is `Agent` really need to be disposable?

Reply:
no and Kosta wasn't sure why... so removing